src/clarity-angular/main.scss

```diff
@@ -7,52 +7,80 @@
 
 /* @echo VERSION */
 
-
+/** Order of imports is important **/
 @import "./utils/overwrites.clarity";
-@import "./utils/mixins.clarity";
-@import "./utils/normalize.clarity";
-@import "./utils/reboot.clarity";
-@import "./utils/helpers.clarity";
-@import "./utils/layers.clarity";
+@import "./utils/layers.clarity"; // no dependencies on other clarity scss
+@import "./utils/helpers.clarity"; // no dependencies on other clarity scss
+@import "./color/color.clarity"; // depends on colors.clarity, helpers.clarity, contrast-cache.clarity
```

Comment:
I think we import it again because there are few files like `variables.clarity.scss` which import `color` but not `mixins`, no?